live/global.ini

```diff
@@ -22294,109 +22294,109 @@ PU_ADVREP01_F_DEH_Enter_StationArea_Resupply_aAlly_tPlayer_FromAlly_FirstOccur_I
-PU_ADVREP01_F_DEH_Enter_Station_HospNotSet_Siege_IG_003_HeresATip=Hier ist ein Tipp, bevor Sie losfahren. Stellen Sie sicher, dass Sie Ihr DNA-Profil auf "Orison General" festgelegt haben, damit Sie schnell wieder mit dem Kampf beginnen können, wenn etwas schief geht.
-PU_ADVREP01_F_DEH_Enter_Station_NFZInactive_Siege_IG_001_WithTheRestricted=Da der eingeschränkte Luftraum wieder unter unserer Kontrolle ist, können Sie Ihr eigenes Schiff anstelle des Shuttles nehmen, wenn Sie möchten. So oder so müssen wir alle verbliebenen Nine Tails und ihre Verteidigungsanlagen säubern, bevor wir die Plattformen Vollständig zurückerobern können.
-PU_ADVREP01_F_DEH_Enter_Station_NFZInactive_Siege_IG_002_JustAHeads=Nur eine Warnung, jetzt, da die Restricted Air unter der Kontrolle der Crusader steht, können Sie Ihr eigenes Schiff nehmen, wenn Sie das Shuttle nicht benutzen möchten. Es gibt immer noch aktive Feinde auf den Plattformen, wir haben also noch einiges zu tun.
-PU_ADVREP01_F_DEH_Enter_Station_NFZInactive_Siege_IG_003_FyiNowThat=Zu Ihrer Information, jetzt, da wir die Kontrolle über den eingeschränkten Luftraum rund um die Plattformen zurückerobert haben, können Sie Ihr eigenes Schiff anstelle des Shuttles nehmen, wenn Sie möchten. Achten Sie einfach auf aktive Luftabwehranlagen.
-PU_ADVREP01_F_DEH_Enter_Station_NoMission_Siege_IG_001_ListenIGet=Hören Sie, ich verstehe, dass Sie sich unbedingt dem Kampf anschließen möchten, aber ich kann nur offiziellen CDF-Freiwilligen erlauben, das Shuttle zu benutzen. Melde dich über Ihren Vertragsmanager an, wenn Sie wirklich daran interessiert sind, zu helfen.
-PU_ADVREP01_F_DEH_Enter_Station_NoMission_Siege_IG_002_SorryThisShuttle=Leider ist dieses Shuttle nur für CDF-Freiwillige. Wir benötigen immer noch Hilfe. Melde dich also gerne über Ihren Vertragsmanager an.
-PU_ADVREP01_F_DEH_Enter_Station_NoMission_Siege_IG_003_EasyThereWe=Einfach da. Wir können in diesem Shuttle nur zertifizierte CDF-Freiwillige mitnehmen. Wenn Sie also helfen möchten, melden Sie sich über Ihren Vertragsmanager an. Andernfalls sollten Sie sich zu Ihrer eigenen Sicherheit von diesem Shuttle fernhalten.
+PU_ADVREP01_F_DEH_Enter_StationArea_Resupply_aAlly_tPlayer_FromAlly_inCombat_FirstOccur_IG_002_YoureAboutTo=Sie werden in Kürze die Station Jericho erreichen, aber wir haben ein Problem. XenoThreat-Kräfte versuchen, die Javelin zu zerstören, die gerade mit Nachschub versorgt wird. Gehen Sie da rein und drängen Sie sie zurück, bevor sie zu viel Schaden anrichten.
+PU_ADVREP01_F_DEH_Enter_StationArea_Resupply_aAlly_tPlayer_FromAlly_inCombat_FirstOccur_IG_003_GladYouMade=Gut, dass Sie es geschafft haben. INS Jericho, die Hauptversorgungsstation der Navy, liegt direkt vor uns, aber die Truppen von XenoThreat haben die Station überfallen und versuchen, die Javelin zu zerstören, die dort angedockt ist. Ich brauche Sie, um dorthin zu gelangen und sie zu verteidigen.
```

Comment:
Ich würde dein langen Satz in 2 kleinere Sätze packen... sind bisschen viele Kommas für den Satz